README.md

```diff
@@ -223,6 +223,8 @@ backends = [
   'retinaface', 
   'mediapipe',
   'yolov8',
+  'yolov11n',
```

Comment:
yolo11s is missing here.

also, you didn't mention the facial recognition models in readme

Reply:
fixed